system/services/assetManager/AssetManagerService.cfc

```diff
@@ -210,13 +211,16 @@ component displayName="AssetManager Service" {
 		  required string  type
 		, required string  size
 		, required string  folderId
+		,          string  assetId
```

Comment:
This argument should either be required or have a default value (""). I think it should probably be required.